src/WebJobs.Script/Utility.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Collections.Specialized;
+using System.Diagnostics;
```

Comment:
Unused?

Reply:
Indeed! From an earlier iteration, thought I got this but must not have, fixed!